src/assets/database.sql

```diff
@@ -366,6 +366,44 @@ INSERT INTO `commands` (`name`, `description`, `examples`, `tool`, `type`, `comm
 ('Zerocopy', '(client only) Use a "zero copy" method of sending data', 'iperf3 --zerocopy', 19, 'checkbox', '--zerocopy', NULL),
 ('Dont fragment', '(client only) Set the IPv4 Dont Fragment (DF) bit on outgoing packets', 'iperf3', 19, 'checkbox', '--dont-fragment', NULL);
 
+--
+-- Populate data for `commands`: dig
+--
+INSERT INTO `commands` (`name`, `description`, `examples`, `tool`, `type`, `command`, `example`, `sudo`, `category`) VALUES 
+('DNS Server', 'Defines the DNS server which will be used in the command', 'dig @8.8.8.8 example.hackinsdn.ubfa,br', 27, 'input', NULL, NULL, NULL, 'BASIC OPTIONS'),
+('DNS domain to query', 'Set the DNS domain which will be queried', 'domain example.hackinsdn.ufba.br', 27, 'input', 'domain', NULL, NULL, 'BASIC OPTIONS'),
+('Query class (default: in)', NULL, 'dig example.com -q-class HS', 27, 'input', '-q-class', NULL, NULL, 'BASIC OPTIONS'),
+('Query type (default: a)', NULL, 'dig example.com -q-type MX', 27, 'input', '-q-type', NULL, NULL, 'BASIC OPTIONS'),
+('Query options, multiple choices', NULL, 'dig example.com -q-opt -4 -b 192.168.1.1#53', 27, 'input', NULL, NULL, NULL, 'BASIC COMMANDS'),
+('Display options, multiple choices', NULL, 'dig example.com +short +dnssec +retry=3 +timeout=5', 27, 'input', NULL, NULL, NULL, 'BASIC COMMANDS'),
+('Global DNS options configuration','Affects all queries', 'dig global-d-opt example.com', 27, 'checkbox', 'global-d-opts', NULL, NULL, 'DNS OPTIONS CONFIGURATIONS'),
+('Local DNS options configurations', 'Affects specific queries', 'dig local-d-opt example.com', 27, 'checkbox', 'local-d-opts', NULL, NULL, 'DNS OPTIONS CONFIGURATIONS');
```

Comment:
Same as the previous comment: from what I know there is no parameters as `global-d-opts` and `local-d-opts` this way, please review it.